queries/nvim/highlights.scm

```diff
@@ -133,9 +142,18 @@
     (constant)
   ] @function.method)
 
+(macro_def
+  name: [
+    (identifier)
+    (constant)
+  ] @function.method)
+
 (param
   name: (identifier) @variable.parameter)
 
+(macro_var
```

Comment:
I personally would prefer the `%` being highlighted the same color, but either way works

Reply:
`%` is included in the identifier token here